packages/core/src/sync/sync-api.ts

```diff
@@ -393,14 +394,35 @@ export class SyncApi implements ISyncApi {
       .subscribe()
   }
 
+  async _getStartBlock(syncOptions: ModelSyncOptions): Promise<number> {
+    let startBlock = Math.max(this.initialIndexingBlock, syncOptions.startBlock || 0)
```

Comment:
what's the difference between `this.initialIndexingBlock` and `syncOptions.startBlock`?